crates/imports/src/std.rs

```diff
@@ -35,14 +39,22 @@ extern "C" {
     fn create_date(value: f64) -> Rid;
 
     #[link_name = "typeof"]
-    fn value_kind(ctx: Rid) -> Kind;
+    pub fn value_kind(ctx: Rid) -> Kind;
```

Comment:
Why is this marked as public?

Reply:
To determine if `Node::next` and `Node::previous` returns `Some(Node)` or `None`:
```
    pub fn next(&self) -> Option<Node> {
        let rid = unsafe { scraper_next(self.0) };
        match unsafe { value_kind(rid) } {
            crate::Kind::Node => Some(Node(rid)),
            _ => None,
        }
    }
```

Reply:
true